content/post/418-Grml-new-stable-release-2024.12-available.md

```diff
@@ -0,0 +1,25 @@
+---
+author: Michael Prokop
+categories: []
+date: Fri, 20 Dec 2024 11:42:23 +0100
+layout: post
+slug: 418-Grml-new-stable-release-2024.12-available
+title: Grml - new stable release 2024.12 available
+
+---
+We are proud to announce the new stable release version 2024.12, code-named ['Adventgrenze'](https://grml.org/faq/#releasename)!
+
+This Grml release provides fresh software packages from Debian trixie.
+As usual it also incorporates current hardware support and fixes known bugs from the previous Grml release.
+
+For the first time ever, version 2024.12 supports 64-bit ARM CPUs (Architecture `arm64`)!
+We would like to thank [netcup](https://www.netcup.com/) for their financial support in making this possible.
+
+At the same time we want to point out that releases for [32-bit x86 PCs have been discontinued]({{< ref "416-grml32-sunset" >}}).
+With the deprecation of the grml32 flavor, we instead implemented UEFI 32-bit boot support on our Grml amd64 flavor.
+
+For detailed information about the changes between 2024.02 and 2024.12 have a look at [the official release announcement](https://grml.org/changelogs/README-grml-2024.12/).
```

Comment:
```suggestion
For a detailed overview of the changes from 2024.02 to 2024.12, please check out [the official release announcement](https://grml.org/changelogs/README-grml-2024.12/).
```

Reply:
ACK, geat suggestions, applied all of them!